src/emailservice/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.9.15-slim as base
+FROM python:3.10.8-slim as base
```

Comment:
**Question:** If we merge this without the hash (base image digest), do you know if Renovate will include the digest when it updates it?

Reply:
Ah that's a good question, I don't know. I just resolved the merged by integrating the digest. Do you want we try with 1 of them, then merge this in `main` and see what will happen? I'm curious to know too, but the only way to see this in action is to merge in `main`. I could open a dedicated PR later if Renovate didn't do it for us to add the digest.

Reply:
Sweet, thanks for including the hashes/digests.

> Do you want we try with 1 of them (without the digest)
I personally don't have a yearning to know since all images now use digests.
But if _you're_ curious, feel free to create a PR. I will approve. :)